docs/install.md

```diff
@@ -16,16 +16,15 @@ You can run Compose on OS X and 64-bit Linux.  It is currently not supported on
 the Windows operating system. To install Compose, you'll need to install Docker
 first.
 
-Depending on how your system is configured, you may require `sudo` access to
-install Compose. If your system requires `sudo`, you will receive "Permission
-denied" errors when installing Compose. If this is the case for you, preface the
-install commands with `sudo` to install.
+> Note: If you get a "Permission denied" error, your `/usr/local/bin` directory
+probably isn't writable and you'll need to install Compose as the superuser. Run
+`sudo -i`, then the two commands above, then `exit`.
```

Comment:
@thaJeztah You are right. The commands had been expanded into steps 4 & 5. Please see revision.

(I find the instructions in step 4 offering multiple ways of installing Compose a little confusing -- even more so after after adding the "Note". Any preference to install Compose with pip vs curl? Should we pick one? Perhaps a separate issue/PR?)